pkg/llms/zhipuai/langchainllm.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2023 KubeAGI.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zhipuai
```

Comment:
Greate!  https://github.com/kubeagi/arcadia/issues/79 This issue can be closed as well. Right ? @Abirdcfly